srcpkgs/fteqw/template

```diff
@@ -0,0 +1,44 @@
+# Template file for 'fteqw'
+pkgname=fteqw
+version=5808
+revision=1
+hostmakedepends="subversion pkgconf"
+makedepends="zlib-devel gnutls-devel libpng-devel libglvnd-devel libXrandr-devel libXcursor-devel alsa-lib-devel libjpeg-turbo-devel opus-devel speex-devel libvorbis-devel freetype-devel"
+short_desc="FTE Quake engine"
+maintainer="Gibson <ypsvlq@gmail.com>"
+license="GPL-2.0-only"
+homepage="https://fte.triptohell.info"
+
+do_fetch() {
+	svn co -r$version svn://svn.code.sf.net/p/fteqw/code/trunk fteqw-$version
+}
+
+do_build() {
+	cd engine
+	if test "$XBPS_MACHINE" == "$XBPS_TARGET_MACHINE"; then
+		make $makejobs qcc-rel sv-rel gl-rel PKGCONFIG=pkgconf
+	else
+		PKG_CONFIG_PATH=$XBPS_CROSS_BASE/lib/pkgconfig make $makejobs qcc-rel sv-rel gl-rel PKGCONFIG=pkgconfig CC=${XBPS_CROSS_BASE#/usr/}-gcc CFLAGS="-I$XBPS_CROSS_BASE/include/freetype2 -I$XBPS_CROSS_BASE/include/opus"
```

Comment:
please split into multiple lines and consider using `make_build_targets` and such to avoid specifying `do_build` directly.